database/src/main/resources/changesets/metadata_journal.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="mcovarr" id="metadata_journal">
+        <createTable tableName="METADATA_JOURNAL">
+            <column autoIncrement="true" name="METADATA_JOURNAL_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_METADATA_JOURNAL"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_UUID" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Sorry if I'm Out of the Loop: Why is this table referring to external string/UUID, instead of a standard foreign key `WORKFLOW_EXECTUION.WORKFLOW_EXECUTION_ID`? Kris mentioned that putting strings/UUIDs as primary keys sometimes causes performance problems. I don't know if it affects other index optimization, but it "smells funny" as we should already have the UUID normalized into an internal WORKFLOW_EXECUTION_ID.

As it's also not listed as a FK, I can only assume it's because for the metadata service we explicitly _do not_ want referential integrity checking? (We'll take ANY UUID, even one not for a workflow!) And/Or, it's also some sort of performance boost by explicitly not reaching into another table for denormalized querying?